src_assets/common/assets/web/public/assets/locale/en.json

```diff
@@ -253,9 +253,9 @@
     "origin_web_ui_allowed_pc": "Only localhost may access Web UI",
     "origin_web_ui_allowed_wan": "Anyone may access Web UI",
     "output_name_desc_unix": "During Sunshine startup, you should see the list of detected displays. Note: You need to use the id value inside the parenthesis.",
-    "output_name_desc_windows": "Manually specify a display to use for capture. If unset, the primary display is captured. Note: If you specified a GPU above, this display must be connected to that GPU. The appropriate values can be found using the following command:",
+    "output_name_desc_windows": "Manually specify a display device id to use for capture. If unset, the primary display is captured. Note: If you specified a GPU above, this display must be connected to that GPU. During Sunshine startup, you should see a list of available display devices and their ids in the log output, e.g.:",
```

Comment:
Can you add to the end like this one? https://github.com/LizardByte/Sunshine/pull/2903#discussion_r1686790388

@gschintgen FYI